CHANGES.md

```diff
@@ -1,5 +1,37 @@
 # Changes
 
+## edge-21.3.3
+
+This release includes various bug fixes and improvements to the CLI, the
+identity and destination control plane components as well as the proxy. This
+release also ships with a new CLI binary for Apple Silicon M1 chips.
+
+* Updated `helm-upgrade` and `upgrade-stable` integration tests now that 2.10
+  has been released
+* Added new RabbitMQ integration tests (thanks @barkardk!)
```

Comment:
We usually don't include info about updated integration tests as they're not user-facing, but we can bend that rule for the RabbitMQ one so we can give back credit :wink: